CustomModule/Crawler.py

```
@@ -11,9 +11,26 @@ def Crawl():
+    elif req.status_code == 200:
+        print("정식(메이저) 갤러리로 인식되었습니다.")
+    else:
+        print("해당 갤러리를 찾을 수 없습니다.")
+        exit()
+
+    page_number = int(input("몇 페이지까지 수집하시겠습니까? (숫자만 입력): ")) # 1페이지부터 몇 페이지까지 수집할 건지 결정
+
+    for page in range(1, page_number + 1):
         # HTTP GET Request
-        response = requests.get("https://gall.dcinside.com/board/lists?id=bike&page={}".format(page), headers=headers)
+        response = requests.get(default_link + "lists?id=" + gall_id +"&page={}".format(page), headers=headers)
```

Ruff (0.14.6)

Probable use of `requests` call without timeout

(S113)